src/odemis/gui/comp/positions.py

```diff
@@ -0,0 +1,254 @@
+
+        fm_fav_pos = stage_md[model.MD_FAV_FM_POS_ACTIVE]
+        sem_fav_pos = stage_md[model.MD_FAV_SEM_POS_ACTIVE]
+        mill_fav_pos = stage_md[model.MD_FAV_MILL_POS_ACTIVE]
+        fib_fav_pos = stage_md[model.MD_FAV_FIB_POS_ACTIVE]
+
+        conf = {"unit": "°", "accuracy": 2}
+
+        pos: dict = {
+            "SEM": sem_fav_pos,
+            "MILL": mill_fav_pos,
+            "FIB": fib_fav_pos,
+            "FM": fm_fav_pos,
+        }
+
+        for name, fav_pos in pos.items():
+            lbl_ctrl = wx.StaticText(self._panel, -1, name)
+            self.gb_sizer.Add(
+                lbl_ctrl,
+                (self.num_rows, 0),
+                flag=wx.ALL | wx.ALIGN_CENTER_VERTICAL,
+                border=5,
+            )
+            rx_val = fav_pos["rx"]
+            rz_val = fav_pos["rz"]
+
+            rx_value_ctrl = UnitFloatCtrl(
+                self._panel, value=math.degrees(rx_val), style=wx.NO_BORDER, **conf
+            )
+            rz_value_ctrl = UnitFloatCtrl(
+                self._panel, value=math.degrees(rz_val), style=wx.NO_BORDER, **conf
+            )
+            self.gb_sizer.Add(
+                rx_value_ctrl,
+                (self.num_rows, 1),  # span=(1, 3),
+                flag=wx.ALL | wx.EXPAND | wx.ALIGN_CENTER_VERTICAL,
+                border=5,
+            )
+            self.gb_sizer.Add(
+                rz_value_ctrl,
+                (self.num_rows, 2),  # span=(3, 5),
+                flag=wx.ALL | wx.EXPAND | wx.ALIGN_CENTER_VERTICAL,
+                border=5,
+            )
+
+            self.ctrl_dict[name] = {"rx": rx_value_ctrl, "rz": rz_value_ctrl}
+
+            rx_value_ctrl.SetForegroundColour(gui.FG_COLOUR_EDIT)
+            rx_value_ctrl.SetBackgroundColour(gui.BG_COLOUR_MAIN)
+            rz_value_ctrl.SetForegroundColour(gui.FG_COLOUR_EDIT)
+            rz_value_ctrl.SetBackgroundColour(gui.BG_COLOUR_MAIN)
+            self.num_rows += 1
+
+        # make text bold, size 12
+        lbl_ctrl = wx.StaticText(self._panel, -1, "Calibration")
+        lbl_ctrl.SetFont(wx.Font(12, wx.DEFAULT, wx.NORMAL, wx.BOLD))
+        self.gb_sizer.Add(
+            lbl_ctrl,
+            (self.num_rows, 0),
+            flag=wx.ALL | wx.ALIGN_CENTER_VERTICAL,
+            border=5,
+        )
+        lbl_ctrl = wx.StaticText(self._panel, -1, "X")
+        self.gb_sizer.Add(
+            lbl_ctrl,
+            (self.num_rows, 1),
+            flag=wx.ALL | wx.ALIGN_CENTER_VERTICAL,
+            border=5,
+        )
+        lbl_ctrl = wx.StaticText(self._panel, -1, "Y")
+        self.gb_sizer.Add(
+            lbl_ctrl,
+            (self.num_rows, 2),
+            flag=wx.ALL | wx.ALIGN_CENTER_VERTICAL,
+            border=5,
+        )
+        self.num_rows += 1
+
+        trans_cor = stage_md[model.MD_POS_TRANS_COR]
+        pos_cor = stage_md[model.MD_POS_COR]
+
+        conf = {"unit": "m", "accuracy": 3}
+
+        pos: dict = {
+            "Rotation": pos_cor,
```

Comment:
It might be of nitpicking, but seeing "rotation", I would expect an angle. But it takes X&Y in meters. Maybe name it "Rotation center" to make this less odd.